ee-9/feature-pack/src/main/resources/modules/system/layers/base/org/apache/activemq/artemis/main/module.xml

```diff
@@ -63,7 +63,6 @@
         <!--this reverse dependency is here so integration classes in the AS code base can be instantiated by Artemis-->
         <module name="org.wildfly.extension.messaging-activemq" services="import"/>
         <module name="org.picketbox" optional="true"/>
```

Comment:
Why do we leave picketbox there ?

Reply:
The PicketBox dependency will be removed in a follow up PR.